frontend/src/components/ProductItem/ProductItem.js

```diff
@@ -1,13 +1,11 @@
 import React from 'react';
-
+import { useCartContext } from '../../context/CartContext'
 import roundedPlusIcon from "../../assets/rounded_plus.svg";
 
-const ProductItem = ({ name, description, price, image }) => {
-  // const productIcons = {
-  //   add: roundedPlusIcon,
-  //   edit: penIcon,
-  //   remove: penIcon,
-  // }
+const ProductItem = ({ id, name, description, category, price, image }) => {
+  const { addProduct } = useCartContext()
+
+  const product = { id, name, description, category, price, image }
```

Comment:
Don't destructure it if you're just gonna re-structure it again – I think it's fine to use the dotted notation in the jsx below, considering it relates to a product and not general props

Reply:
Yeah you're right. Fixed!